### DISCUSSION TITLE
Quickstart의 <NaverMapsProvider />와 <NavermapsProvider />

### DISCUSSION THREAD

**Discussion by postforty** [zeakd/react-naver-maps#116] [General]
이 소스를 공유해주셔서 매우 감사한 마음입니다 👍👍
Quickstart의 소스 코드를 적용하던 중에 다음과 같은 에러를 만났습니다.

Attempted import error: 'NaverMapsProvider' is not exported from 'react-naver-maps' (imported as 'NaverMapsProvider').
ERROR in ./src/App.js 9:30-47  

"react-naver-maps"를 까보니
`export { Props as NaverMapsProviderProps, NavermapsProvider } from './provider.js';`
는 있는데 NaverMapsProvider와 관련된 것은 없었습니다.

또한 제공해 주신 문서를 찾아보니 [Migration Guide](https://zeakd.github.io/react-naver-maps/guides/migration-guide-from-0.0/)에 다음의 내용이 있었습니다.

"기존 RenderAfterNavermapsLoaded 는 SSR 문제와 컴포넌트 렌더 시의 모호함이 있었고 v0.1 에서 제거되었습니다. 대신, Application Root 에 `<NavermapsProvider />`를 추가해주세요"

결론적으로 말씀드리면 NaverMapsProvider가 아니라 NavermapsProvider가 맞다면 Quickstart 페이지에도 반영하는 것이 이 라이브러리를 이용하시는 분들께 오해가 없지 않을까 하는 생각이 들어 글을 작성하게 되었습니다.

제 글을 읽어 주시고, "react-naver-maps"를 오픈해 주심에 다시금 감사드립니다.

**Comment by zeakd** [comment]
네 맞습니다. https://github.com/zeakd/react-naver-maps/pull/107 이러한 내용이고 시간날 때 문서 작업해두겠습니다 🙏
